elixir/apps/web/lib/web/live/settings/api_clients/beta.ex

```diff
@@ -25,16 +25,19 @@ defmodule Web.Settings.ApiClients.Beta do
       <:title><%= @page_title %></:title>
       <:help>
         API Clients are used to manage Firezone configuration through a REST API. See our
-        <a class={link_style()} href="https://api.firezone.dev/swaggerui">interactive API docs</a>
+        <a class={link_style()} target="_blank" href="https://api.firezone.dev/swaggerui">
```

Comment:
We should probably get my other PR merged in before this, because this is fixed in that PR.

Reply:
https://github.com/firezone/firezone/pull/6202

Reply:
K I'll rebase and handle conflicts after CI is fixed